### PR TITLE
reorganize README.md

### PR DESCRIPTION
**Pull request by mnecas**
reorganize README.md for better readability dont mix role options with engine options

Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1772096
@mwperina

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,24 +41,29 @@ By default engine-setup uses answer file specific for version of oVirt,
...
+| ovirt_engine_setup_version            | 4.2                   | Allowed versions: [4.1, 4.2] |
+| ovirt_engine_setup_package_list       | []                    | List of extra packages to be installed on engine apart from ovirt-engine package. |
 | ovirt_engine_setup_fqdn               | UNDEF                 | Host fully qualified DNS name of the server. |
 | ovirt_engine_setup_organization       | UNDEF                 | Organization name for certificate. |
 | ovirt_engine_setup_firewall_manager   | firewalld             | Specify the type of firewall manager to configure on Engine host, following values are availableL: `firewalld`,`iptables` or empty value to skip firewall configuration. |
-| ovirt_engine_setup_update_setup_packages | False              | If `True`, setup packages will be updated before `engine-setup` will be executed. Makes sense if Engine is already installed. |
-| ovirt_engine_setup_update_all_packages | True                 | If `True`, all packages will be updated before `engine-setup` will be executed. |
 | ovirt_engine_setup_offline            | False                 | If `True`, `engine-setup` will not search for package updates. |
```
Comment (mwperina) [inline README.md]: I think this one should be moved to common options

File: README.md [hunk 2]
```diff
@@ -41,24 +41,29 @@ By default engine-setup uses answer file specific for version of oVirt,
 based on ``ovirt_engine_setup_version`` parameter. You can specify own answer file
 as ``ovirt_engine_setup_answer_file_path``.
 
-* Common options:
+* General options for role:
```
Comment (mwperina) [inline README.md]: Common options for the role:

File: README.md [hunk 3]
```diff
@@ -41,24 +41,29 @@ By default engine-setup uses answer file specific for version of oVirt,
...
 | ovirt_engine_setup_answer_file_path   | UNDEF                 | Path to custom answerfile for `engine-setup`. |
 | ovirt_engine_setup_use_remote_answer_file | false             | If `True`, use answerfile's path on the remote machine. This option should be used if the installation occure on the remote machine and the answerfile is located on there also. |
-| ovirt_engine_setup_fqdn               | UNDEF                 | Host fully qualified DNS name of the server. |
-| ovirt_engine_setup_organization       | UNDEF                 | Organization name for certificate. |
-| ovirt_engine_setup_firewall_manager   | firewalld             | Specify the type of firewall manager to configure on Engine host, following values are availableL: `firewalld`,`iptables` or empty value to skip firewall configuration. |
 | ovirt_engine_setup_update_setup_packages | False              | If `True`, setup packages will be updated before `engine-setup` will be executed. Makes sense if Engine is already installed. |
+| ovirt_engine_setup_perform_upgrade    | False                 | If true this role is used to perform upgrade. |
```
Comment (mwperina) [inline README.md]: Let's be consistent and use `True` everywhere

File: README.md [hunk 4]
```diff
@@ -41,24 +41,29 @@ By default engine-setup uses answer file specific for version of oVirt,
...
 | ovirt_engine_setup_update_all_packages | True                 | If `True`, all packages will be updated before `engine-setup` will be executed. |
+| ovirt_engine_setup_product_type       | oVirt                 | One of ["oVirt", "RHV"], case insensitive. |
 | ovirt_engine_setup_offline            | False                 | If `True`, `engine-setup` will not search for package updates. |
+
+* Common options for engine:
+
+| Name                            | Default value         |  Description                                              |
+|---------------------------------|-----------------------|-----------------------------------------------------------|
+| ovirt_engine_setup_version            | 4.2                   | Allowed versions: [4.1, 4.2] |
```
Comment (mwperina) [inline README.md]: Shouldn't the default be 4.4?

Comment (mnecas) [reply README.md]: Yes there is merge conflict with #71

File: README.md (hunk 3 above)
Comment (mnecas) [reply README.md]: I don't see the reason for `True` also multiple other varialbes have default value `False`.

Comment (mnecas) [reply README.md]: Okay I'll update the description

**Pull request merged**